libyul/backends/evm/EVMDialect.cpp

```diff
@@ -383,6 +390,7 @@ std::vector<std::optional<BuiltinFunctionForEVM>> createBuiltins(langutil::EVMVe
 			1,
 			1,
 			SideEffects{},
```

Comment:
One more thing:

```suggestion
			EVMDialect::sideEffectsOfInstruction(evmasm::Instruction::DATALOADN),
```

I think the result will be the same, but it should really be coming from `SemanticInformation` whenever possible to avoid mistakes.

Reply:
fixed